### PR TITLE
Feature/objects

### PR DESCRIPTION
**Pull request by Noah-Stalder**

### CODE REVIEW

File: Backend/src/main/java/com/coderbunker/supqr/database/FeedbackRepository.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@Injectable
+public class FeedbackRepository extends AbstractRepository {
+
+	@Inject
+	public FeedbackRepository (DSLContextFactory factory) {
+		super(factory);
+	}
+
+	public List<Feedback> getRatingOfArticle (Integer articleId) {
+
```
Comment (jsauvain) [inline Backend/src/main/java/com/coderbunker/supqr/database/FeedbackRepository.java]: remove new line

File: Backend/src/main/java/com/coderbunker/supqr/database/FeedbackRepository.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		List<Feedback> feedbacks = getContext()
+			.select()
+			.from(FEEDBACK)
+			.where(FEEDBACK.ARTICLE_ID.eq(articleId))
+			.fetchInto(Feedback.class);
+
+		return feedbacks;
+
```
Comment (jsauvain) [inline Backend/src/main/java/com/coderbunker/supqr/database/FeedbackRepository.java]: remove new line

File: Backend/src/main/java/com/coderbunker/supqr/database/FeedbackRepository.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	public List<Feedback> getRatingOfArticle (Integer articleId) {
+
+		List<Feedback> feedbacks = getContext()
+			.select()
+			.from(FEEDBACK)
+			.where(FEEDBACK.ARTICLE_ID.eq(articleId))
+			.fetchInto(Feedback.class);
+
+		return feedbacks;
```
Comment (jsauvain) [inline Backend/src/main/java/com/coderbunker/supqr/database/FeedbackRepository.java]: return directly don't store it

File: Backend/src/main/java/com/coderbunker/supqr/database/ObjectRepository.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+@Injectable
+public class ObjectRepository extends AbstractRepository {
+
+	@Inject
+	public ObjectRepository (DSLContextFactory factory) {
+		super(factory);
+	}
+
+	public List<Article> getArticlesByUserId (int userId) {
+
```
Comment (jsauvain) [inline Backend/src/main/java/com/coderbunker/supqr/database/ObjectRepository.java]: remove new line

File: Backend/src/main/java/com/coderbunker/supqr/rest/model/ObjectSummaryTO.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+ * Copyright (c) Ergon Informatik AG, Switzerland
+ */
+
+package com.coderbunker.supqr.rest.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
```
Comment (jsauvain) [inline Backend/src/main/java/com/coderbunker/supqr/rest/model/ObjectSummaryTO.java:13]: add @AllArgsConstructor and @NoArgsConstructor

File: Backend/src/main/java/com/coderbunker/supqr/rest/model/ObjectTo.java [hunk 6]
```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) Ergon Informatik AG, Switzerland
+ */
+
+package com.coderbunker.supqr.rest.model;
+
+public class ObjectTo {
```
Comment (jsauvain) [inline Backend/src/main/java/com/coderbunker/supqr/rest/model/ObjectTo.java]: Should this be pushed?

File: Backend/src/main/java/com/coderbunker/supqr/rest/model/RatingTO.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+ */
+
+package com.coderbunker.supqr.rest.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class RatingTO {
```
Comment (jsauvain) [inline Backend/src/main/java/com/coderbunker/supqr/rest/model/RatingTO.java:16]: @AllArgsConstrucotr @NoArgs..

File: Backend/src/main/java/com/coderbunker/supqr/rest/model/RatingTo.java [hunk 8]
```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) Ergon Informatik AG, Switzerland
+ */
+
+package com.coderbunker.supqr.rest.model;
+
+public class RatingTo {
+}
```
Comment (jsauvain) [inline Backend/src/main/java/com/coderbunker/supqr/rest/model/RatingTo.java]: Should this be pushed?

File: Backend/src/main/java/com/coderbunker/supqr/rest/resource/ObjectResource.java [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class ObjectResource {
+
+	private final ObjectService objectService;
+
+	@GET
+	@Path("/me")
+	public List<ObjectSummaryTO> getMyObjects () {
+		int id = 2;
```
Comment (jsauvain) [inline Backend/src/main/java/com/coderbunker/supqr/rest/resource/ObjectResource.java]: Mark as temporary with a comment

**Review by jsauvain — CHANGES_REQUESTED** [review]

File: Backend/src/main/java/com/coderbunker/supqr/rest/model/ObjectTo.java (hunk 6 above)
Comment (Noah-Stalder) [reply Backend/src/main/java/com/coderbunker/supqr/rest/model/ObjectTo.java]: niemert weiss?

**Review by jsauvain — APPROVED** [review]

**Pull request merged**